style.css

```diff
@@ -1,59 +1,975 @@
 @import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic:wght@400;700;800&display=swap');
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
 
-body{
-    font-family:'Nanum Gothic', sans-serif;
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Nanum Gothic', sans-serif;
+  font-weight: 400;
+  margin: 0;
+  color: #797575;
+  scroll-behavior: smooth;
 }
 
 nav {
+  display: flex;
+  flex-direction: row;
+  gap: 15%;
+  position: fixed;
+  z-index: 20;
+  height: 5%;
+  width: 100vw;
+  padding: 20px;
+  top: 0%;
+  background-color: white;
+}
+
+@media (max-width: 768px) {
+  .ul-nav {
+    position: fixed;
+    left: -100%;
     display: flex;
+    top: 0;
+    padding-top: 0;
+    gap: 10px;
+    flex-direction: column;
+    background: rgb(39, 42, 50);
+    opacity: 0.95;
+    width: 70%;
+    height: 100vh;
+    font-size: 27px;
+    margin-top: 0;
+    transition: 0.9s;
+    list-style-type: none;
+  }
+
+  .ul-nav li p {
+    margin-top: 0;
+    color: #f7f1f1;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  button {
+    border-style: none;
+    font-size: 40px;
+    background: rgb(39, 42, 50);
+    opacity: 0.95;
+    color: #f7f1f1;
+    display: flex;
+    justify-content: flex-end;
     flex-direction: row;
-    justify-content: space-between;
-    position: fixed;
-    height: 20%;
-}
-.microverse-logo {
+    padding-right: 40px;
+  }
+
+  .ul-nav.active {
+    left: 0;
+  }
+
+  /* .microverse-logo {
```

Comment:
- [ ] I kindly suggest that your remove commented out code from your codebase to make your work neater.